arraycontext/impl/pytato/__init__.py

```diff
@@ -235,6 +236,24 @@ def get_target(self):
 
 # {{{ PytatoPyOpenCLArrayContext
 
+
+@dataclass
+class ProfileEvent:
+    """Holds a profile event that has not been collected by the profiler yet."""
+
+    cl_event: cl._cl.Event
+    translation_unit: Any
```

Comment:
Do we want the `translation_unit` in here? I'm not sold.

Reply:
Changed it to just the name of the translation unit in [7216ccf](https://github.com/inducer/arraycontext/pull/311/commits/7216ccf34e16cb390170d5251848f90eb2787406)